apc/proxy.php

```diff
@@ -9,12 +9,35 @@
 	}
 }
 
+/**
+ * Determines if SSL is used.
+ *
+ * @see is_ssl() (wp-includes/load.php).
+ *
+ * @return bool True if SSL, otherwise false.
+ */
+function cachify_is_ssl() {
+	if ( isset( $_SERVER['HTTPS'] ) ) {
+		if ( 'on' == strtolower( $_SERVER['HTTPS'] ) ) {
```

Comment:
I had seen, that its a copy from the WP source, but maybe this function inside the core is also not the best. My point for this strict coding is performance. The `==` coerces or converts the data type temporarily. The strict comparison is much faster and if we can get sure, that the string is right, then we should use the strict comparison.
My two cents, maybe the other will get more values on your reason, there is also valid.